src/support_sphere/lib/presentation/components/auth/login_form.dart

```diff
@@ -13,19 +13,32 @@ class LoginForm extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BlocListener<LoginCubit, LoginState>(
-      listenWhen: (previous, current) => previous.status != current.status,
-      listener: (context, state) {
-        if (state.status.isFailure) {
-          ScaffoldMessenger.of(context)
-            ..hideCurrentSnackBar()
-            ..showSnackBar(
-              const SnackBar(
-                content: Text('Authentication Failure'),
-              ),
-            );
-        }
-      },
+    return MultiBlocListener(
+      listeners: [
+        BlocListener<LoginCubit, LoginState>(
+          listenWhen: (previous, current) => 
+              previous.email != current.email ||
+              previous.password != current.password,
+          listener: (context, state) {
+            context.read<LoginCubit>().setValid();
+            context.read<LoginCubit>().validateAllFieldsFilled();
```

Comment:
Once the `isAllFieldsFilled` is turned into a computed state, probably don't even need this function anymore.

Reply:
updated!